pysph/base/kernels.py

```diff
@@ -89,31 +89,41 @@ def kernel(self, xij=[0., 0, 0], rij=1.0, h=1.0):
 
         return val * fac
 
-    def gradient(self, xij=[0., 0, 0], rij=1.0, h=1.0, grad=[0, 0, 0]):
+    def sigmadwdq(self, rij=1.0, h=1.0):
```

Comment:
Why the term `sigmadwdq`?  Can't you just call this `dwdq`?

Reply:
I can call dwdq, initially I did that, but the actual formula reads as

```
DWIJ = normalization_factor * dw_dq * (x_ij / (r_ij * h))
```
So keep it clear I named it as `sigmadwdq`

Reply:
No, this is not standard, in that case return dw_dq.  There is no reference to any formula here.  It would be nice to have a docstring or to update the documentation on this.  There is no standard convention on what "sigma" is.  So please change this.

Reply:
Will this get fixed by just adding the documentation, or should I change the name to something else.

In many papers they represent normalization factor by sigma, so I used that notation.

I will change the function name to `dwdq` and write a doc string about what it is.